modules/mono/editor/GodotTools/GodotTools/GodotTools.csproj

```diff
@@ -30,7 +30,15 @@
     <ConsolePause>false</ConsolePause>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="JetBrains.Annotations, Version=2019.1.3.0, Culture=neutral, PublicKeyToken=1010a0d8d6380325">
+      <HintPath>..\packages\JetBrains.Annotations.2019.1.3\lib\net20\JetBrains.Annotations.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
     <Reference Include="Mono.Posix" />
+    <Reference Include="Newtonsoft.Json, Version=12.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed">
+      <HintPath>..\packages\Newtonsoft.Json.12.0.3\lib\net45\Newtonsoft.Json.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
```

Comment:
This may be automated in the future but for now this assembly needs to be added here in order for it to be distributed when building Godot: https://github.com/godotengine/godot/blob/8eb183aebb9c79ff92d6f566af7ad2f91696ce08/modules/mono/build_scripts/godot_tools_build.py#L83-L86

Reply:
Is it also required to call a special `nuget_restore` as called here?
https://github.com/godotengine/godot/blob/master/modules/mono/build_scripts/godot_tools_build.py#L54

Reply:
No, it's called here: https://github.com/godotengine/godot/blob/435cced7d8fbc80e6842d7c171a87d4db1e64fe4/modules/mono/build_scripts/godot_tools_build.py#L17

The other one is a special case for building only the `GodotTools.ProjectEditor` project instead of the whole solution (this is done when running SCons with `mono_glue=no`).